### ISSUE TITLE
Panda and Main Screen image replacement

### ISSUE THREAD

**Issue by philchfree** [bigtreetech/PandaTouch#67]
The change of the Panda image works great in FW1.03 but can we have the same for the Main Screen Logo. BIGTREETECH is already engraved at the bottom of the screen, I'd like to be able to put my own logo on the main screen. professionnals will want to have the same change as well for sure to put their own logo.

**Comment by DoTsTeR77** [comment]
I agree.. or even the option to set that logo seperately would be cool..

**Comment by crobin75** [comment]
Yes this is a needed feature. Also the need to support Mac for picture updating not just windows

**Comment by Uzuniita** [comment]
hi all, im trying to set the image but didnt work for me. just update to 1.0.3, resized to 280x306, pixel depth 32-bit and generate a log it says "max 202x202 and i have done too but when upload didnt change anything

**Comment by philchfree** [comment]
Resize it to 250x250

**Comment by fatalpunx** [comment]
I tried converting to .img,says file size to small,how do you change the big tree tech logo,I followed step by step instructions to change the .img file but never works,I can rename the original file anything I want reload that just fine,but anything else with the disc image format won't work?

**Comment by philchfree** [comment]
It's not the bigtreetech logo that gets modified byt the panda image while printing. You will only see your change if you actually start a print

**Comment by xperiments** [comment]
Closing this issue as stale due to inactivity.
